planner/core/testdata/integration_suite_out.json

```diff
@@ -336,12 +336,11 @@
       {
         "SQL": "desc select /*+ INL_HASH_JOIN(t2) */ distinct t2.a from t t1 join t t2 on t1.a = t2.a",
         "Plan": [
-          "HashAgg_8 8000.00 root  group by:test.t.a, funcs:firstrow(test.t.a)->test.t.a",
-          "└─IndexHashJoin_20 12500.00 root  inner join, inner:TableReader_13, outer key:test.t.a, inner key:test.t.a",
-          "  ├─TableReader_22(Build) 10000.00 root  data:TableFullScan_21",
-          "  │ └─TableFullScan_21 10000.00 cop[tikv] table:t1 keep order:false, stats:pseudo",
-          "  └─TableReader_13(Probe) 1.00 root  data:TableRangeScan_12",
-          "    └─TableRangeScan_12 1.00 cop[tikv] table:t2 range: decided by [test.t.a], keep order:false, stats:pseudo"
+          "IndexHashJoin_17 12500.00 root  inner join, inner:TableReader_10, outer key:test.t.a, inner key:test.t.a",
```

Comment:
Why this agg can be eliminated?
Because t1.a and t2.a both are unique?

Reply:
Yes, `t2.a` is a unique key in this level schema.